### PR TITLE
Dot env configuration

### PR DESCRIPTION
**Pull request by jac1013**
#4

### CODE REVIEW

File: .env-template [hunk 1]
```diff
@@ -0,0 +1 @@
+WEB_SERVER_PORT=9000
```
Comment (rkmax) [inline .env-template:1]: Is kinda silly but it's best on the template set the same value as the default, 3000

Comment (jac1013) [reply .env-template:1]: My intention is to show that when people uses this as the `.env` file they will see the application running in a different port (which is feedback that the configuration is working), what do you think? if you still think that we need to use the same just make a suggestion and I'll accept it, if you make the suggestion please approved this so I can merge it :sweat_smile:

File: api/todo.go [hunk 2]
```diff
@@ -23,15 +30,19 @@ func NewTodoRestConfigurator(handler TodoOperations) {
...
-	router.GET("/todo/:id", todoOperations.readTodo)
+	router.POST("/todo", todoRestConfig.createTodo)
+	router.GET("/todo/:id", todoRestConfig.readTodo)
 	// Content-Type of "application/json" must be used for this endpoint handler
-	router.PUT("/todo", todoOperations.updateTodo)
-	router.DELETE("/todo/:id", todoOperations.deleteTodo)
+	router.PUT("/todo", todoRestConfig.updateTodo)
+	router.DELETE("/todo/:id", todoRestConfig.deleteTodo)
 
 	// By default it serves on :8080 unless a
```
Comment (DiegoSabalsa) [inline api/todo.go]: By default it serves on 3000

Comment (DiegoSabalsa) [inline api/todo.go]: By default it serves in 3000

Comment (DiegoSabalsa) [inline api/todo.go]:
```suggestion
	// By default it serves on :3000 unless a
```

**Review by DiegoSabalsa — CHANGES_REQUESTED** [review]

**Review by DiegoSabalsa — APPROVED** [review]

**Pull request merged**